src/vitabel/timeseries.py

```diff
@@ -3151,9 +3289,10 @@ def mouse_click_listener(event: MouseEvent):
                                         active_label.intervals[interval_index, :]
                                     )
                                     repaint_plot(start, stop)
+                               
                                 return
 
-                            if partial_interval_data is None:
+                            elif partial_interval_data is None:
```

Comment:
No need to change into `elif` thanks to the `return` in the block above

```suggestion
                            if partial_interval_data is None:
```